spec/controllers/games_controller_spec.rb

```diff
@@ -25,4 +25,25 @@
       expect(response).to have_http_status(:success)
     end
   end
+
+  describe 'games#index action' do
+    it "should succesfully show the page" do
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'should give an overview of games with only one player' do
+      game1 = FactoryGirl.create(:game)
+      game2 = FactoryGirl.create(:game)
+      game3 = FactoryGirl.create(:game)
+      player1 = FactoryGirl.create(:player, game_id: game1.id)
+      player2 = FactoryGirl.create(:player, game_id: game2.id)
```

Comment:
Absolutely. I thought rubocop wanted me to delete the lines altogether, but this makes total sense. I'll add a commit to clean it up